### DISCUSSION TITLE
confused by vertical align in button text

### DISCUSSION THREAD

**Discussion by yureckey** [primefaces/community#61] [PrimeVue]
I wouldn't call myself a perfectionist, but when I'm looking at primeVue buttons I cannot unsee that its vertical alignment isn't centered.
What is even stranger is that when I downloaded Stackblitz project and build it - vertical alignment was even more "non-centered" than in the docs (I didn't changed code, only updated libs versions in package.json). Here is screenshot with center lines added in the Figma:
![primevue_button_vertical](https://github.com/primefaces/community/assets/535466/1748c4c7-bb7a-467d-9761-b1790cc8fab9)

**Comment by tugcekucukoglu** [comment]
Lara's font was changed to Inter-var. Could you check again please?

**Reply by yureckey** [reply]
Honestly, it looks same to me (the font is different, the label position is same)
![image](https://github.com/primefaces/community/assets/535466/0f555f1b-8b6f-47b1-aac3-625b4a898bc3)
I don't know, maybe it's not that big deal and I'll get used to it

**Reply by cagataycivici** [reply]
@onursenture @dcyaner What do you guys think? Do we have an alignment issue?

**Reply by onursenture** [reply]
I can't see a problem, also I don't understand what is being attempted to be conveyed with the drawn alignment lines.

**Reply by yureckey** [reply]
Well, the lines are drawn in the exact vertical middle of the button rectangle. As you can see, text is not divided equally by this line, but lower part of text consumes more space than top part - hence it appears as not aligned.
BUT
I just realized, that I wasn't comparing exactly the same buttons in Quasar and Vuetify. These libraries implement material design which by default have button labels in CAPS. When I checked Quasar and Vuetify buttons with caps disabled, then it shows the same "not quite equal" behavior. And vice versa, PrimeVue button with Caps shows label aligned vertically just fine. So, since everybody showing this behavior I won't try to persuade libraries developers that it should be fixed.
Sorry for wasting your time.
![image](https://github.com/primefaces/community/assets/535466/581dd333-daac-4ec0-8cea-b6ccc199f950)

**Comment by SoyDiego** [comment]
I don't know if the same problem or related but for example I'm using Linux and the alignment I can see different if I compare with Windows.

Here for example I can see more space in the padding bottom:
![image](https://github.com/primefaces/community/assets/19764334/909de996-b1f0-4217-a103-01bc99aa2933)

And here for example I can see the letters a little bit more in the left side:
![image](https://github.com/primefaces/community/assets/19764334/707709cb-08dd-4be3-bdb4-49ea53bec2e3)

But If I'm in Windows I can see all perfectly centered and aligned

In the beginning I was thinking is something wrong but then I realized was Linux and the browser xD
